packages/Button/src/LinkButton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+// import iconButtonStyles from "./IconButton.styles";
+import Button from "./Button";
+
+const LinkPropTypes = {
+  children: PropTypes.node.isRequired,
+  kind: PropTypes.oneOf(Button.Kinds.ALL),
+  size: PropTypes.oneOf(ShirtSizes.MEDIUM),
+  url: PropTypes.string,
+};
+
+const LinkDefaultProps = {
+  kind: Button.Kinds.DEFAULT,
+  size: ShirtSizes.MEDIUM,
+  url: null,
+};
+function redirect_to(url) {
+  window.location = url;
+}
+const LinkButton = React.forwardRef((props, ref) => {
+  const buttonProps = {
+    children: null,
+    icon: props.children,
+    isFullWidth: false,
+  };
+  // debugger/
+  // buttonProps.children = <a href={props.url} />
+  return (
+    <Button
```

Comment:
We actually want to render a `<a>` tag. This is because we don't want to use `window.location`. It isn't good for accessibility.

https://webaim.org/techniques/hypertext/

I think the challenge here is being able to render a <a> but still make it look like a <Button>.